File: Python/計實作業/P3-110502557.py
'''
Practice 03
Name: 游宗穎
Student Number: 110502557
Course: 2021-CE1003-B
'''

def create_file(file_name, file_context):
    file = open(file_name, "w")
    file.write(file_context)
    print("File name: " + file_name)
    print("Context: " + file_context)
    file.close()

create_file(input("Create a file: "), input("Write something: "))

File: Java/計實作業/P3_110502557.java
/**
 * Practice 03
 * Student Number: 110502557
 * Course: 2022-CE1004-A
 */

public class P3_110502557 {
    public static void main(String[] args) {
        // 以下不要改
        Student putin = new Student("Putin", 59);
        System.out.println(putin.getName() + " before: " + putin.getScore());
        putin.changeScore(40);
        System.out.println(putin.getName() + " after: " + putin.getScore());
//        putin.score = 100;
    }
}

class Student {
    private String name;
    private int score;

    public Student(String name, int score) {
        this.name = name;
        this.score = score;
    }

    public String getName() {
        return name;
    }

    public int getScore() {
        return score;
    }

    public void changeScore(int newScore) {
        score = newScore;
    }
}